recipes/quickstart/Getting_to_know_Llama.ipynb

```diff
@@ -1 +1,1677 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "ioVMNcTesSEk"
+   },
+   "source": [
+    "##**0 - Prerequisites**\n",
+    "* Basic understanding of Large Language Models\n",
+    "\n",
+    "* Basic understanding of Python"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {
+    "executionInfo": {
+     "elapsed": 248,
+     "status": "ok",
+     "timestamp": 1695832228254,
+     "user": {
+      "displayName": "Amit Sangani",
+      "userId": "11552178012079240149"
+     },
+     "user_tz": 420
+    },
+    "id": "ktEA7qXmwdUM"
+   },
+   "outputs": [],
+   "source": [
+    "# presentation layer code\n",
+    "\n",
+    "import base64\n",
+    "from IPython.display import Image, display\n",
+    "import matplotlib.pyplot as plt\n",
+    "\n",
+    "def mm(graph):\n",
+    "  graphbytes = graph.encode(\"ascii\")\n",
+    "  base64_bytes = base64.b64encode(graphbytes)\n",
+    "  base64_string = base64_bytes.decode(\"ascii\")\n",
+    "  display(Image(url=\"https://mermaid.ink/img/\" + base64_string))\n",
+    "\n",
+    "def genai_app_arch():\n",
+    "  mm(\"\"\"\n",
+    "  flowchart TD\n",
+    "    A[Users] --> B(Applications e.g. mobile, web)\n",
+    "    B --> |Hosted API|C(Platforms e.g. Custom, HuggingFace, Replicate)\n",
+    "    B -- optional --> E(Frameworks e.g. LangChain)\n",
+    "    C-->|User Input|D[Llama 3]\n",
+    "    D-->|Model Output|C\n",
+    "    E --> C\n",
+    "    classDef default fill:#CCE6FF,stroke:#84BCF5,textColor:#1C2B33,fontFamily:trebuchet ms;\n",
+    "  \"\"\")\n",
+    "\n",
+    "def rag_arch():\n",
+    "  mm(\"\"\"\n",
+    "  flowchart TD\n",
+    "    A[User Prompts] --> B(Frameworks e.g. LangChain)\n",
+    "    B <--> |Database, Docs, XLS|C[fa:fa-database External Data]\n",
+    "    B -->|API|D[Llama 3]\n",
+    "    classDef default fill:#CCE6FF,stroke:#84BCF5,textColor:#1C2B33,fontFamily:trebuchet ms;\n",
+    "  \"\"\")\n",
+    "\n",
+    "def llama2_family():\n",
+    "  mm(\"\"\"\n",
+    "  graph LR;\n",
+    "      llama-2 --> llama-2-7b\n",
+    "      llama-2 --> llama-2-13b\n",
+    "      llama-2 --> llama-2-70b\n",
+    "      llama-2-7b --> llama-2-7b-chat\n",
+    "      llama-2-13b --> llama-2-13b-chat\n",
+    "      llama-2-70b --> llama-2-70b-chat\n",
+    "      classDef default fill:#CCE6FF,stroke:#84BCF5,textColor:#1C2B33,fontFamily:trebuchet ms;\n",
+    "  \"\"\")\n",
+    "\n",
+    "def llama3_family():\n",
+    "  mm(\"\"\"\n",
+    "  graph LR;\n",
+    "      llama-3 --> llama-3-8b\n",
+    "      llama-3 --> llama-3-70b\n",
+    "      llama-3-8b --> llama-3-8b-base\n",
+    "      llama-3-8b --> llama-3-8b-instruct\n",
+    "      llama-3-70b --> llama-3-70b-base\n",
```

Comment:
Same here. Lets remove base

Reply:
Should we ask HF to remove base also then? https://huggingface.co/blog/llama3#whats-new-with-llama-3

Reply:
I don't see them using the word base in the naming. They explain it after the :. Also the models are named like this https://huggingface.co/meta-llama/Meta-Llama-3-8B on HF

Reply:
Ah I see - the "-" in "llama-3-8b-base" is not meant to be the model name, but added there to make that mm method happy (replacing - with space would break the mm call). 
Anyway, happy to make you happy - i'l get rid of "base" in the names.